### DISCUSSION TITLE
How can I collect predicted values?

### DISCUSSION THREAD

**Discussion by umichwbaek** [mir-group/allegro#35] [Q&A] [answered]
Hi,

I want to get all the predicted energies and force of evaluation data from 'nequip-train' model. The 'nequip-evaluate' only shows statistical metrics like mae and rmse. I was wondering if there have a function or command to do.

**Comment by Linux-cpp-lisp** [comment, marked as answer]
`nequip-evaluate`'s `--output` and `--output-fields` options allow outputing an extXYZ file containing arbitrary model outputs over whatever dataset you've configured `nequip-evaluate` to run over.
